pkg/reconciler/knativeserving/knativeserving.go

```diff
@@ -74,8 +72,8 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, original *servingv1alpha1
 		}
 	}
 
-	manifest, err := r.transform(ctx, original)
-	if err != nil {
+	manifest := r.config.Append()
```

Comment:
Might want to comment that this is a `Clone()` operation essentially.

Reply:
Done